packages/teams-js/test/public/liveShare.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { LiveShareHost } from '../../src/internal/liveShareHost';
+import { liveShare } from '../../src/public/liveShare';
+
+describe('MicrosoftTeams-LiveShare', () => {
+  describe('initialize', () => {
```

Comment:
You need some unit tests that verify that the "correct" path works (Calling initialize doesn't throw any errors)

Reply:
See the previous comment about `live-share` would need to be published which hasn't happened yet

Reply:
Removed namespace per discussion